### DISCUSSION TITLE
Clarification on the Profile Type in User search API

### DISCUSSION THREAD

**Discussion by vinod303** [Sunbird-Lern/Community#145] [Query] [answered]
Hi Team,
I have noticed that, when we do a user search api, i am seeing the following two fields as below for profile types.  Hence seeking clarity on which one is the correct one / or both are used ? 

**API**: 
`/api/user/v3/search`

**RESULT**
   "content": [
                {
                    "maskedPhone": "******6360",
                    "rootOrgName": "XXXXXXXX",
                    "channel": "XXXXXXXXXXX",
                    "profileUserTypes": [
                        {
                            "type": "student"
                        }
                    ],
                    
                    "profileUserType": {
                        "subType": null,
                        "type": "student"
                    },
                    
    Note: I am hitting this api in SB 4.6
    @rhwarrier @AmiableAnil @Jayaprakash8887

**Comment by Hari-stackroute** [comment, marked as answer]
Please do search with "profileUserTypes" @vinod303

**Reply by Jayaprakash8887** [reply]
@vinod303 Please let us know if your query is answered.

**Reply by vinod303** [reply]
@Jayaprakash8887 @Hari-stackroute  thanks
